### DISCUSSION TITLE
How to disable YouTube Shorts in the new version?

### DISCUSSION THREAD

**Discussion by ShrimpBFM** [TeamNewPipe/NewPipe#10681] [Questions] [answered]
I just updated NewPipe to 0.26.0.
I was my first update in months.

But I immediately noticed: 
I'm now seeing YouTube shorts again in my subscription feed. 🤢
And I can't find any option to disable them in the settings.

**Does anyone know how to get rid of shorts in my subscription feed and my search results?**

I don't remember if this app used to have a setting to disable them, or if they were just not supported before.
All I know is, those were the good times :P

Thanks for any help.

(As you might guessed from this post, I really don't like seeing shorts clogging up my feeds. 
To me they are time wasting TikTok garbage that floods my subscription feed and search result, making it harder to find/see actual YouTube videos. 
I use a combination of extensions and filters on PC, just so I never have to see them there either. 😈)

**Comment by petlyh** [comment, marked as answer]
You can disable shorts in the feed by unchecking 'Shorts' in Content > Feed > Fetch channel tabs.
l'm not aware of any way to hide shorts from search results.

**Reply by ShrimpBFM** [reply]
Thank you!
I guess the update reset that option for me, and then I forgot where the setting was :)

**Reply by misieck** [reply]
im on 0.26.1. That setting doesnt seem to work. Still getting shorts in whats new feed.
In fact i unchecked all the options for the feed tabs and it still gets new content!

**Reply by opusforlife2** [reply]
Try the dedicated feed toggle.

**Reply by misieck** [reply]
Im not sure what is that "dedicated" to?
Either case I noticed i dont get shorts when not using the fast mode for content updates. I had it turned off for a while now and had no shorts. As soon as i turn the fast mode on again, I get shorts.

**Reply by opusforlife2** [reply]
Right now, Shorts don't seem to have an upload date, so the feed fetcher skips over them, except for the fast feed. We don't have a solution for this yet.

**Reply by michaelachrisco** [reply]
Could we create a search filter based on length of videos? It's basically just YouTube shorts now coming up as search results.

**Reply by DUOLabs333** [reply]
Is there a way to block Shorts in results, but _not_ in the feed?
